README.md

```diff
@@ -454,6 +454,23 @@ To use the JAR with no dependencies in a Java application, the following require
 ### Building and using the Javadoc JAR to extract the Javadoc HTML files (amazon-timestream-jdbc-\<version\>-javadoc.jar)
 Javadoc JAR builds with `mvn install` alongside the other JAR files. To extract the Javadoc HTML files, use the following command: `jar -xvf amazon-timestream-jdbc-1.0.0-javadoc.jar`
 
+### Debug with BI tools
+#### DBeaver
+To enable debug logs, the application should be run with `-Ddbeaver.trace.enabled=true` flag
```

Comment:
Let's add more examples, similar to what DBeaver suggests - https://github.com/dbeaver/dbeaver/wiki/JDBC-Tracing

As an example - where to enable it, complete instructions for the user (e.g. restart DBeaver), where to find log files, the name of log files, and an example of the log generated by the driver.